python/plugins/processing/gui/BatchInputSelectionPanel.py

```diff
@@ -23,9 +23,16 @@
 from pathlib import Path
 
 from qgis.PyQt.QtCore import pyqtSignal, QCoreApplication
-from qgis.PyQt.QtWidgets import QWidget, QHBoxLayout, QMenu, QPushButton, QLineEdit, QSizePolicy, QAction, QFileDialog
+from qgis.PyQt.QtWidgets import QWidget, QHBoxLayout, QMenu, QPushButton, QLineEdit, QSizePolicy, QFileDialog
 from qgis.PyQt.QtGui import QCursor
 
+# TODO QGIS 4.0: Fix code if PyQt6
+from qgis.PyQt.QtCore import QT_VERSION
+if (QT_VERSION >= 0x060000):
+    from qgis.PyQt.QtGui import QAction
+else:
+    from qgis.PyQt.QtWidgets import QAction
```

Comment:
Could you make QAction available from qgis.PyQt.QtGui for the PyQt5 version instead?
I believe I've done that for Qt4 as well a while back

Reply:
Done [e4a7acc](https://github.com/qgis/QGIS/pull/52739/commits/e4a7acc8b7d4c579b93b55269b10c8ae25497972)